Source/Controller/NotificationsViewController.swift

```diff
@@ -128,72 +141,91 @@ class NotificationsViewController: ContentViewController {
 
     private func update(with reports: [Report], animated: Bool = true) {
         self.dataSource.reports = reports
+        lastTimeNewReportsUpdatesWasChecked = Date()
         self.tableView.reloadData()
+        self.navigationController?.tabBarItem?.badgeValue = nil
+    }
+
+    func checkForNotificationUpdates(force: Bool = false) {
+        if force {
```

Comment:
Maybe when the user taps on this tab (i.e. viewDidAppear) that would be another good time to do a forced check?

Reply:
It check for notification updates each time we create a report, so it will be already checking for new reports when the user landed on that screen.